WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -3372,6 +3374,12 @@ textFieldCssClass:"admin_xform_number_input"}
 									txtBoxLabel:ZaMsg.LBL_zimbraEnforcePwdHistory, labelLocation:_LEFT_, 
 									textFieldCssClass:"admin_xform_number_input",
 									visibilityChecks:[],enableDisableChecks:[[ZaAccountXFormView.isAuthfromInternalSync, entry.name, ZaAccount.A_name]]
+								},{ref:ZaAccount.A_zimbraPasswordBlockCommonEnabled, type:_SUPER_CHECKBOX_, 
+									resetToSuperLabel:ZaMsg.NAD_ResetToCOS, 
+									msgName:ZaMsg.NAD_RejectCommonPwd,
+									checkBoxLabel:ZaMsg.NAD_RejectCommonPwd, 
+								 	trueValue:"TRUE", falseValue:"FALSE",
+									visibilityChecks:[],enableDisableChecks:[[ZaAccountXFormView.isAuthfromInternalSync, entry.name, ZaAccount.A_name]]
```

Comment:
what does isAuthfromInternalSync check do?

Reply:
According to my understanding by observing the helper functions, it seems a flag which represent if the authentication made by zimbra internally or by external parties.
We are keeping some flags disabled if the authentication made by external parties for preferences related to Password-management.
So i did use the same.

You can share your thoughts here, we will act accordingly.